examples/nlp/parameter_efficient_finetuning_of_gpt2_with_lora.py

```diff
@@ -448,6 +456,8 @@ def call(self, inputs):
     # Change query dense layer.
     decoder_layer = lora_model.backbone.get_layer(f"transformer_layer_{layer_idx}")
     self_attention_layer = decoder_layer._self_attention_layer
+    # Allow mutation to Keras layer state.
+    self_attention_layer._tracker.locked = False
```

Comment:
Not optimal (not actionable here though). We should either make this a public API, or think of a better way to do this (preferable).